artifacts/mysql-statefulset.yaml

```diff
@@ -66,6 +79,11 @@ spec:
           subPath: mysql
         - name: conf
           mountPath: /etc/mysql/conf.d
+        {{ if ne .Spec.FromBackup.BackupName "" }}
```

Comment:
I'm not sure if all `containers` need the backup volume mounted, `initContainers` do the job of restoring the data.

Reply:
You're right, we only need that in mysql-init.